operations/boot_orchestration/Rolling_Upgrades.md

```diff
@@ -25,6 +25,8 @@ For more information on staging sessions, see [Creating a Staged Session](./Stag
 1. The workload manager calls the `applystaged` endpoint for each node when it is ready.
 BOS then copies the information staged for these components into their desired state, and BOS starts to operate on the nodes and attempts to make their actual state match their new desired state.
 
+<a name="staged-sessions-with-slurm"></a>
```

Comment:
I think @haasken-hpe call this out in another PR, but these sorts of entries are not necessary (or even allowed by the repo checks?).  You can just reference the section by name in Markdown without adding anything special like this.

Reply:
I had removed it from main and thought the cherry-pick to 1.4 had gotten it, but I was wrong.
Will remove.